Makefile

```diff
@@ -2,7 +2,7 @@ docker/cloud/build:
 	docker build -f docker/cloud/Dockerfile -t go-nitro-cloud .
 docker/cloud/start:
```

Comment:
Outside the scope of this PR but should this make command also start the other cloud nodes?